### PR TITLE
sanity: cleanup of policy.py

### PR DESCRIPTION
**Pull request by kkrolczyk**
checking policy-report -crf ... -lpf ... -> requires those args

in case of missing files - traceback was displayed, now usr info
in case of improper JSON - traceback, now usr info
in case of missing props - traceback, on KeyErrors, now usr info

Signed-off-by: Krzysztof Królczyk <Krzysztof.Krolczyk@o2.pl>

### CODE REVIEW

File: flict/__main__.py [hunk 1]
```diff
@@ -266,6 +266,11 @@ def suggest_outbound_candidate(args):
 
 
 def policy_report(args):
+    try:
+        open(args.report_file).close()
+        open(args.policy_file).close()
+    except (FileNotFoundError):
+        flict_exit(ReturnCodes.RET_FILE_NOT_FOUND, "Provided file was not found.")
```
Comment (priv-kweihmann) [inline flict/__main__.py]: It would be indeed nice to actually let the user know which file is not accessible

File: flict/impl.py [hunk 2]
```diff
@@ -87,8 +87,12 @@ def verify(self):
...
 
     def _read_compliance_report(self, report_file):
-        with open(report_file) as fp:
-            return json.load(fp)
+        on_error = "Internal error, report file has improper format, required: JSON"
+        try:
+            with open(report_file) as file_:
+                return json.load(file_)
+        except json.decoder.JSONDecodeError:
+            raise FlictError(ReturnCodes.RET_INTERNAL_ERROR, on_error)
```
Comment (priv-kweihmann) [inline flict/impl.py:95]: Could you add some unit tests please - just so we have this covered and avoiding future regressions

**Review by priv-kweihmann — CHANGES_REQUESTED** [review]
See review comments.
Main concern is the missing tests.

Other than that, this is a very welcome change

File: flict/impl.py (hunk 2 above)
Comment (kkrolczyk) [reply flict/impl.py:95]: sure,  added

File: flict/__main__.py (hunk 1 above)
Comment (kkrolczyk) [reply flict/__main__.py]: done

**Review by priv-kweihmann — APPROVED** [review]

**Pull request merged**